test/e2e/operator_test.go

```diff
@@ -180,6 +180,175 @@ func TestDefaultDNSSteadyConditions(t *testing.T) {
 	}
 }
 
+func TestDNSLogging(t *testing.T) {
+	cl, err := getClient()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Ensure that DNS is stable before starting the test, otherwise we'd need to tweak individual test durations.
+	if err := waitForDNSConditions(t, cl, 5*time.Minute, dnsName, defaultAvailableDNSConditions...); err != nil {
```

Comment:
Added this to ensure that DNS is stable before starting the test. This cuts down on the number of individual waits that need to be tweaked.